data/reusables/actions/runner-name-description.md

```diff
@@ -1 +1 @@
-The name of the runner executing the job.
+The name of the runner executing the job. Also the name may not be unique in a workflow run as runners at repository and organization level could use the same name.
```

Comment:
I'd probably suggest not using `Also`. My default preference would be `Note that`, but I'd defer to the docs owners.

Reply:
[]()